### ISSUE TITLE
Criteria for grouping of TX groups

### ISSUE THREAD

**Issue by magnolia61** [Project60/sepa_dd#228]
Also in to organize our workflow and keep track we would like to split the mandates in different groups so we set up different payment processors for donations and event registration.

Would it be possible to have it be configurable to split / group the mandates in the way is best suited for the organisation.

I can imagine us doing a montly run for our donations, two different fixed collection dates for our events and a fixed yearly date for membership fees. For these type I would like different TX groups (also because or the different collection dates), so also different creditors.

But to manage our events easily it would help us greatly if we could have a different TX group per event, so can have more grip on the totals and processing refused contributions.

The bottom line of all this is obviously that us tech and functional thinking people make the lived of our volunteers handling the payments easy and doable :-) Any smart and helpfull thoughts on the grouping?

**Comment by Detsieber** [comment]
I would approach this by setting up different creditors - which might
all point to the same bank account...

Am 13.01.2015 um 00:17 schrieb Richard van Oosterhout:

**Comment by systopia** [comment]
@Detsieber  is absolutely correct, that's how it is intended.

@magnolia61: please close if satisfactory.

**Comment by magnolia61** [comment]
I can live with this. 
If I want different TX groups for each eventtype I need to make a lot of different creditor ID's, but it is doable. Thanks for thinking along.
